docs/pages/desktop-access/troubleshooting.mdx

```diff
@@ -255,6 +255,33 @@ $ certutil -pulse
 
 ## Connection attempts fail
 
+### RDP Server only uses standard RDP Security
```

Comment:
```suggestion
### RDP server only uses standard RDP security
```

Reply:
Standard RDP Security and Enhanced RDP Security are the actual proper names: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/8e8b2cca-c1fa-456c-8ecb-a82fc60b2322

Does this change your mind or should I still make the change?